microsoft/testsuites/vm_extensions/vmsnapshot_extension.py

```diff
@@ -47,6 +56,151 @@ class VmSnapsotLinuxBVTExtension(TestSuite):
+        supported or unsupported for selective billing feature.
+        The test would be passed in both the cases, just that the information helps in
+        clearly classifying the distro, when the test runs on various distros.
+        """,
+        priority=2,
+        requirement=simple_requirement(
+            supported_features=[AzureExtension],
+        ),
+    )
+    def verify_exclude_disk_support_restore_point(
+        self,
+        log: Logger,
+        node: Node,
+        environment: Environment,
+        result: TestResult,
+    ) -> None:
+        # Any extension will do, use CustomScript for convenience.
+        # This test would only run when extension is installed already else will
+        # triggere a retsore point that helps in installing the extension on the VM
+        # Copy the local files into the VM.
+        # Move the files to the extension directory
+        # Give the execution permissions to the files
+        # Run the files
+        # Validate the results
+        extension_dir = self._find_extension_dir(node)
+        if extension_dir == "":
+            self._verify_vmsnapshot_extension(log, node, environment)
+            trial = 0
+            while trial < 3:
+                time.sleep(2)
+                extension_dir = self._find_extension_dir(node)
+                if extension_dir == "":
+                    trial = trial + 1
+                else:
+                    break
+
+        # installing all the required packages
+        posix_os: Posix = cast(Posix, node.os)
+        posix_os.install_packages("python2.7")
+        # install pip
+        url = "https://bootstrap.pypa.io/pip/2.7/get-pip.py"
+        args = "-o get-pip.py"
+        curl = Curl(node)
+        curl.fetch(url=url, arg=args, shell=True, execute_arg="", sudo=True)
+        node.execute(cmd="python2.7 get-pip.py", shell=True, sudo=True)
+        node.execute(cmd="python2.7 -m pip --version")
+        # install mock module
+        node.execute(cmd="python2.7 -m pip install mock", sudo=True)
+        # copy the file into the vm
+        self._copy_to_node(node, "handle.txt")
+
+        if extension_dir != "":
+            # moving the handle.py file to extension directory
+            script = (
+                "#!/bin/sh\n"
+                f"mv {node.working_path}/handle.txt {extension_dir}/handle.py"
+            )
+            script_base64 = base64.b64encode(bytes(script, "utf-8")).decode("utf-8")
+            settings = {"script": script_base64}
+            extension = node.features[AzureExtension]
+            extension.create_or_update(
+                name="CustomScript",
+                publisher="Microsoft.Azure.Extensions",
+                type_="CustomScript",
+                type_handler_version="2.1",
+                auto_upgrade_minor_version=True,
+                settings=settings,
+            )
+            log.info(extension_dir)
```

Comment:
Add descriptions with the variable. or it's hard to understand from the log only.